scripts/database.js

```diff
@@ -249,3 +249,15 @@ const database = {
     },
   ],
 };
+
+export const getSchools = () => {
+  return database.schools.map(schools => ({ ...schools }))
+}
+
+export const getArchetypes = () => {
+  return database.archetypes.map(archetypes => ({ ...archetypes}))
+}
+
+export const getBooks = () => {
+  return database.books.map(books => ({ ...books }))
```

Comment:
The iterator should be singular, not plural, as it represents one book object in the database.books array.